(async function() {
    try {
    } catch(e) {
        console.error(e);
    }
})();